pages/garden/index.js

```diff
@@ -336,28 +373,48 @@ function Garden({ petCollection, onInteractPet }) {
             {currentPet.picture}
           </PetDisplay>
         </PetWrapper>
-        <ListPageLink>
-          <Link href="/pet-list">List</Link>
-          {/* update link to list page once replaced as mainpage required */}
-        </ListPageLink>
+        <AdjustedListPageLink>
+          <Link
+            href="/pet-list"
+            role="img"
+            aria-label="Staple of Books indicating List"
+          >
+            📚
+          </Link>
+        </AdjustedListPageLink>
+        <AdjustedDetailPageLink>
+          <Link
+            href={{
+              pathname: `/pet-details/${currentPet.id}`,
+              query: {
+                health: currentPet.status.health,
+                happiness: currentPet.status.happiness,
+                hunger: currentPet.status.hunger,
+                energy: currentPet.status.energy,
+                intelligence: currentPet.status.intelligence,
+              },
+            }}
+            role="img"
+            aria-label="Magnifying Glass indicating Details"
+          >
+            🔎
+          </Link>
```

Comment:
I know it's a bit confusing with the "roles" and the "aria-labels etc". But I'll try to explain it again.  ⬇️

- Using `role="img"` on a `Link` with an emoji as its content isn't the best approach. The `role="img"` should only be used when an element is meant to behave like an image, and it's typically paired with `alt` text for accessibility. 
- For Links with emojis, a better approach is to use `aria-label` without `role="img"` to provide an accessible description for screen readers. The `aria-label `will tell users what the emoji represents without treating the emoji as an image.

➡️ **So remove `role="img"` and keep the `aria-label` on every link.**